lib/view_component/form.rb

```diff
@@ -1,10 +1,20 @@
 # frozen_string_literal: true
 
 require "view_component"
+require_relative "form/configuration"
```

Comment:
Do we need this line? Can Zeitwerk autoload the constant without it?

Reply:
I ran the test suite without it in d1e420c and it seems to be loaded just fine!

I'll do another test in my production Rails application.